app/src/main/java/com/ivyclub/contact/ui/main/add_friend/ExtraInfoListAdapter.kt

```diff
@@ -0,0 +1,45 @@
+package com.ivyclub.contact.ui.main.add_friend
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.ivyclub.contact.R
+import com.ivyclub.contact.databinding.ItemAddFriendExtraInfoBinding
+import com.ivyclub.contact.util.binding
+
+class ExtraInfoListAdapter :
+    ListAdapter<FriendExtraInfoData, ExtraInfoListAdapter.ExtraInfoViewHolder>(DIFF_UTIL) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ExtraInfoViewHolder {
+        return ExtraInfoViewHolder(parent.binding(R.layout.item_add_friend_extra_info))
+    }
+
+    override fun onBindViewHolder(holder: ExtraInfoViewHolder, position: Int) {
+        holder.bind(getItem(position))
+    }
+
+    inner class ExtraInfoViewHolder(val binding: ItemAddFriendExtraInfoBinding) :
```

Comment:
```suggestion
    class ExtraInfoViewHolder(private val binding: ItemAddFriendExtraInfoBinding) :
```
외부 클래스 참조하는 것이 없어서 `inner` modifier 빼고, binding을 클래스 내부에서만 접근해서 `private` 접근 제한자를 추가했습니다!